src/components/home/Team/index.js

```diff
@@ -6,34 +6,64 @@ import { StaticQuery, graphql } from "gatsby"
 
 import "./index.module.css"
 
-const socialLink = (platform, username) => {
+const socialDetails = (platform, username) => {
   switch (platform) {
     case "twitter":
-      return `https://twitter.com/${username}`
+      return {
+        linkName: "Twitter",
+        linkShortname: "Tw",
+        url: `https://twitter.com/${username}`,
+      }
+    case "dribbble":
+      return {
+        linkName: "Dribble",
+        linkShortname: "Dri",
+        url: `https://dribbble.com/${username}`,
+      }
+    case "github":
+      return {
+        linkName: "Github",
+        linkShortname: "Git",
+        url: `https://github.com/${username}`,
+      }
     default:
-      return "https://subvisual.co"
+      return {
+        linkName: "Other",
+        linkShortname: "Other",
+        url: "https://subvisual.co",
```

Comment:
I still think this is not a great default value.

Reply:
Why?

Reply:
1. Because `#` won't even cause a reload;
2. If you forgot to add the information for any social network, this won't warn you of anything, you'll have to get it by looking at the final result, where an error would warn a developer right away;
3. While this should redirect to the same page, it is an assumption not being made anywhere else;
4. We'll change to subvisual.com as our main domain with this rebranding.